package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Shared module for FxA repositories",
   "main": "index.js",
   "scripts": {
-    "test": "mocha --recursive test"
+    "test": "mocha --recursive test",
+    "lint": "eslint email/** l10n/** test/**"
```

Comment:
is this going to miss https://github.com/mozilla/fxa-shared/blob/master/index.js ?

Can we do something like `**/*.js` or `*.js`